components/02-elements/wrapper-collapsible/_wrapper-collapsible.scss

```diff
@@ -0,0 +1,71 @@
+.wrapper-collapsible {
+    display: block;
+    overflow: hidden;
+
+    &__container {
+        display: none;
+        position: absolute;
+        top: 50px;
+        right: 0;
+        width: auto;
+        min-width: 260px;
+        background-color: $white;
+        z-index: 10;
+        overflow: hidden;
+
+        @include mq($screen-l) {
+            top: 95px;
+        }
+
+        &.open {
+            display: block;
+        }
+    }
+
+    &__mobile {
```

Comment:
Nah... Do not use anything like `desktop`, `tablet`, `mobile`, `phone` or anything that may describe any particular device type. 

I.e. `desktop` isn't desktop anymore while you are using page on TV or 17' laptop 😉